README.md

```diff
@@ -44,6 +44,9 @@ add_library(mylib ...)
 #  Python module dependencies (requirements.txt content)
 # SCRIPTS
 #  Additional scripts that should be part of the wheel.
+# PYTHON_PACKAGE_DIRS
+#  Additional python projects which could be wrappers around the
```

Comment:
I would rephrase this: "Paths to directories of additional Python packages, which are bundled with the wheel. This could for example contain pybind11 binding module wrappers."

Reply:
Okay. did that. However I am using it with Boost.Python not pybind11. I guess it doesn't matter.

Reply:
Awesome, good to know. Thanks!